org.jacoco.core.test.validation.kotlin/src/org/jacoco/core/test/validation/kotlin/targets/KotlinWhenExpressionTarget.kt

```diff
@@ -30,8 +30,8 @@ object KotlinWhenExpressionTarget {
     @Suppress("REDUNDANT_ELSE_IN_WHEN")
     private fun whenSealedRedundantElse(p: Sealed): Int = when (p) { // assertFullyCovered()
         is Sealed.Sealed1 -> 1 // assertFullyCovered(0, 2)
-        is Sealed.Sealed2 -> 2 // assertFullyCovered(1, 1)
-        else -> throw NoWhenBranchMatchedException() // assertNotCovered()
+        is Sealed.Sealed2 -> 2 // assertFullyCovered(0, 0)
+        else -> throw NoWhenBranchMatchedException() // assertEmpty()
```

Comment:
Why don‘t we show instructions any more on this line? Or does this compile to something that our filter cannot distinguish from compiler generated code?

Reply:
@marchof Unfortunately without reliance on line numbers implicit `NoWhenBranchMatchedException` for exhaustive when-expression indistinguishable from explicit when using  Kotlin compiler version `1.5`.